docs/lib/analytics/fragments/ios/getting-started.md

```diff
@@ -73,23 +78,3 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
     return true
 }
 ```
-
-## Using a Custom Plugin
-
-You can create your custom plugin for Analytics. This may be helpful if you want to integrate your app with a custom analytics backend.
-
-```swift
- import AmplifyPlugins
-
- ...
- 
- func getEscapeHatch() throws {
-    let plugin = try Amplify.Analytics.getPlugin(for: "awsPinpointAnalyticsPlugin")
-    guard let pinpointAnalyticsPlugin = plugin as? AWSPinpointAnalyticsPlugin else {
-        XCTFail("Could not get plugin of type AWSPinpointAnalyticsPlugin")
-        return
-    }
-    let awsPinpoint = pinpointAnalyticsPlugin.getEscapeHatch()
-    XCTAssertNotNil(awsPinpoint)
-}
```

Comment:
I was thinking escape hatch documentation should live at the end (on the last section/page for v2-preview docs). is this is removed completely?

Reply:
no, it's still there. It was duplicated in the Getting Started too, I just removed this one

Reply:
Good call. The escape hatch definitely doesn't belong in the getting started section.